docs/faq.md

```diff
@@ -61,6 +61,20 @@ environment. `minikube delete` will reset the cluster state.
 
 ---
 
+How do I...?
+---------------------
+
+### Q: How do I configure my app dynamically?
+
+The [Tiltfile API](api.html) has multiple escape hatches for reading configuration:
```

Comment:
I dunno to what extent this is nitty or philosophical, but I don't think of any of these as escape hatches. Do you? Maybe "features", or "has multiple ways to read configuration"?

Reply:
ya, fair! I changed it to "several built-in functions"